### PR TITLE
Implement bid and unbid

### PR DESCRIPTION
**Pull request by danielSbastos**
Hey there,

In this PR, the bid and unbid functionality are implemented. Apart from this, some minor changes regarding the interaction with the keyring are altered, i.e., its events were brought back to the kusama context file in order to correctly call some methods.

### After bid was done, now clicked to unbid
Screenshot from 2022-02-05 20-31-27

### Waiting for unbid
Screenshot from 2022-02-05 20-31-39

### Bid list
image

### CODE REVIEW

File: src/pages/explore/BiddersPage/index.tsx [hunk 1]
```diff
@@ -1,27 +1,62 @@
...
   const [bids, setBids] = useState<Vec<PalletSocietyBid> | []>([])
+  const [result, setResult] = useState(null)
+  const [showAlert, setShowAlert] = useState(true)
 
   const loading = !api?.query?.society
 
   useEffect(() => {
     if (!loading) {
-      api.query.society.bids().then((response: Vec<PalletSocietyBid>) => {
+      api.query.society.bids((response: Vec<PalletSocietyBid>) => {
```
Comment (danielSbastos) [inline src/pages/explore/BiddersPage/index.tsx:22]: By removing `then` and instead passing an unsubscribe function, we are effectively subscribing to changes on the bid list.

**Comment by danielSbastos** [comment]
[Implement the bid/vouch component](https://trello.com/c/bkCRRD2V/278-implement-the-bid-vouch-component)

File: src/pages/explore/BiddersPage/BidVouch.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+type BidVouchProps = { handleResult: any, activeAccount: accountType }
+
+const BidVouch = ({ handleResult, activeAccount } : BidVouchProps) => {
+  const { api } = useKusama()
+  const [bidAmount, setbidAmount] = useState(0)
+  const [loading, setLoading] = useState(false)
+
+  useEffect(() => {
+    const bid = async () => {
+      const bid = api?.tx?.society?.bid(bidAmount)
```
Comment (laurogripa) [inline src/pages/explore/BiddersPage/BidVouch.tsx:17]: WDYT of chaning this to always check for `api` and `apiIsReady`?

**Review by laurogripa — APPROVED** [review]
Looking good!

**Pull request merged**